dagshub/data_engine/model/datapoint.py

```
@@ -173,25 +193,25 @@ def get_blob(self, column: str, cache_on_disk=True, store_value=False) -> bytes:
-                raise RuntimeError(f"Error while downloading blob: {content}")
 
             if store_value:
                 self.metadata[column] = content
             elif cache_on_disk:
                 self.metadata[column] = blob_location
 
             return content
+        elif isinstance(current_value, MetadataAnnotations):
+            return current_value.to_ls_task()
         else:
             raise ValueError(f"Can't extract blob metadata from value {current_value} of type {type(current_value)}")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)